### DISCUSSION TITLE
Rep malus on damage/destroyed power line

### DISCUSSION THREAD

**Discussion by Daek-wml** [Vdauphin/HeartsAndMinds#1567] [Question and answer] [answered]
Hello, Currently using v1.23.5 on Tanoa. 

How do I change or remove power line from being tagged as a building? It's causing massive rep malus during Ops. Currently I just set building rep malus to zero. Thanks

**Comment by Vdauphin** [comment, marked as answer]
Hello,
Why is it specific to Tanoa?

May want t look here: https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/mission.sqf#L302
You can add something like `["Wire", 0]`

Cheers

**Reply by Daek-wml** [reply]
I haven't used vanilla maps in a while. Thought it might be relevant, didn't have this problem on CUP maps.

Adding ["Wire", 0] works, Thanks!

**Reply by Vdauphin** [reply]
I think it is more related to the fact that Tanoa have a lot of power line even inside city

> Adding ["Wire", 0] works, Thanks!

You welcome!
